### DISCUSSION TITLE
build error when using jest

### DISCUSSION THREAD

**Discussion by bodao0405** [quasarframework/quasar#10406] [Quasar App Extensions]
1. npm install -g @quasar/cli
2. quasar create project
3. quasar dev
works well.
After added jest, build error:
4. quasar ext add @quasar/testing-unit-jest
5. quasar dev
![image](https://user-images.githubusercontent.com/18025553/129661355-e140d546-ec9a-4efc-aa53-caba6c16c1a7.png)

**Comment by ontwikkelfabriek** [comment]
@bodao0405 

Can you post the output of `quasar info`?

**Comment by bodao0405** [comment]
I ignored the important info,  quasar-jest depends on vue2.x, but there's only vue3 in my environment。 How can i resolve it?
**
npm WARN @vue/test-utils@1.2.2 requires a peer of vue@2.x but none is installed. You must install peer dependencies yourself.
npm WARN @vue/test-utils@1.2.2 requires a peer of vue-template-compiler@^2.x but none is installed. You must install peer dependencies yourself.
npm WARN vue-jest@3.0.7 requires a peer of vue@^2.x but none is installed. You must install peer dependencies yourself.
npm WARN vue-jest@3.0.7 requires a peer of vue-template-compiler@^2.x but none is installed. You must install peer dependencies yourself.
**

**Comment by bodao0405** [comment]
another issue:  
"quasar ext remove @quasar/testing-unit-jest"  did not delete test or configuration files.

**Reply by ontwikkelfabriek** [reply]
@bodao0405 

Testing is not officially ready for Qv2

There's a beta for Qv2:

 `quasar ext add  @quasar/testing-unit-jest@alpha`


See:
https://github.com/quasarframework/quasar-testing/releases

**Comment by erhard** [comment]
OK - Thank I installed the alpha version but now the test-utils are not fitting to the examples. Are there recipes out there how to test with quasar 2......Or any doku ?    
Thanks for that great work of quasar guys !
